src/Runner.php

```diff
@@ -764,7 +764,7 @@ public function processFile($file)
      * The reporting information returned by each child process is merged
      * into the main reporter class.
      *
-     * @param array $childProcs An array of child processes to wait for.
+     * @param array<non-negative-int, non-empty-string> $childProcs An array of child processes to wait for.
```

Comment:
```suggestion
     * @param array<int, string> $childProcs An array of child processes to wait for.
```

These annotation should not be tooling specific and `non-negative-int` and `non-empty-string` are PHPStan specific annotations.